src/Ryujinx.Ava/UI/Windows/CheatWindow.axaml.cs

```diff
@@ -27,12 +28,12 @@ public CheatWindow()
             Title = $"Ryujinx {Program.Version} - " + LocaleManager.Instance[LocaleKeys.CheatWindowTitle];
         }
 
-        public CheatWindow(VirtualFileSystem virtualFileSystem, string titleId, string titleName)
+        public CheatWindow(VirtualFileSystem virtualFileSystem, string titleId, string titleName, string gameBuildId)
```

Comment:
Pass the file path instead and call `ApplicationData.GetApplicationBuildId()` here.

On a side note:
```suggestion
        public CheatWindow(VirtualFileSystem virtualFileSystem, string titleId, string titleName, string buildId)
```